public/js/monitor.js

```diff
@@ -127,7 +127,12 @@ function removeInvalidMessage(e) {
 }
 
 function doOauth() {
-  window.open("/oauth/init");
+  if (localStorage.getItem("scanned")) {
+    window.open(`/oauth/init/?scanned=${localStorage.getItem("scanned")}`);
```

Comment:
Do we need to `encodeURIComponent()` this (in case it is an email with a `+` or something)?

Reply:
Will do. Closing this PR in favor of opening one for the fxa-updates branch and will make this update there.